frontend/src/pages/EmailVerificationPage.jsx

```
@@ -0,0 +1,112 @@
+          Verify Your Email
+        </h2>
+        <p className="mb-6 text-center text-gray-300">
+          Enter the 6-digit code sent to your email.
+        </p>
+
+        <form onSubmit={handleSubmit} className="space-y-6">
+          <div className="flex justify-between">
+            {code.map((digit, index) => (
+              <input
+                key={index}
+                ref={(el) => (inputRefs.current[index] = el)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)